src/index.pug

```diff
@@ -78,8 +80,11 @@ html(lang='en')
           .insights__subline Climate Mind consists of a web application and a knowledge base.
           .insights__text The finished product will guide users through the process of personalizing climate conversations with others and responding to common climate change myths. Then it will provide achievable action proposals that align with the values and interests of the user.
           .insights__text The knowledge base will collect knowledge available on climate change and interpret it as sets of concepts and the relationships between them.
+          a.button(href="https://app.climatemind.org/" target="_blank") Go to the App
       .insights__wrapper
-        img.insights__image(src="images/app-preview.jpg" alt="ClimateMind app")
+        <a href="https://app.climatemind.org/" target="_blank">
```

Comment:
Just a minor hint: you could use the Pug syntax here and make line 87 obsolete. But both works, whichever you prefer.
`a(href="https://app.climatemind.org/" target="_blank")`

Reply:
good to know, thanks!